### PR TITLE
Game logic branch

### PR DESCRIPTION
**Pull request by Brianified**
added websocket game logic

### CODE REVIEW

File: src/server/service/draft_service.py [hunk 1]
```diff
@@ -1,38 +1,106 @@
...
+        self.pack_size = draft_params.getPackSize()
+        self.time_limit = draft_params.getRoundTime()
+        self.player_decks = {}
+        self.player_e_decks = {}
+        self.player_m_packs = {}
+        self.player_e_packs = {}
+        self.player_picked = [0 for i in range(self.max_players)]
+        self.num_packs = None
+
+    def createDraft(self):
```
Comment (SeraphShroud) [inline src/server/service/draft_service.py]: rename to create_draft()

File: src/server/models/draft_params.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+class DraftParams(object):
+    """description of class"""
+    def __init__(self):
+        self.main_list = []
+        self.extra_list = []
+        self.player_length = 0
+        self.round_time = 0
+        self.pack_size = 0
+
+    def setMainList(self, main_list):
```
Comment (SeraphShroud) [inline src/server/models/draft_params.py]: rename all functions to have _ such as set_main_list

File: src/server/tests/test_draft_service.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+path = os.path.dirname(os.path.abspath(__file__))
+# parent directory 
+parent = os.path.dirname(path) 
+import sys
+sys.path.append(parent)
+from service.draft_service import DraftService
+from models.draft_params import DraftParams
+import unittest
+
+draftParams = DraftParams()
```
Comment (SeraphShroud) [inline src/server/tests/test_draft_service.py:14]: rename all camelcase strings to _

**Review by SeraphShroud — CHANGES_REQUESTED** [review]
Functionality to add:
1) When in a draft waiting lobby, everyone should be able to see everyone who is in the lobby, not just the people joining after them. Mockup page: https://app.moqups.com/fNeestatg8/view/page/a3dc6978f
2) Create an endpoint to game handler that retrieves all lobbies (open and mid-draft) with their respective room info so client can create a join lobby. Mockup page: https://app.moqups.com/fNeestatg8/view/page/a808f4dc3
3) Fix ygo_test unit test

File: src/server/tests/test_draft_service.py (hunk 3 above)
Comment (Brianified) [reply src/server/tests/test_draft_service.py:14]: yeah... I'm not gonna do that its too much

**Pull request merged**